lib/guard-rspec-view.coffee

```diff
@@ -15,7 +15,7 @@ class GuardRspecView
     terminal = spawn("bash", ["-l"])
 
     projectPath = atom.project.getPaths()[0]
-    command = "guard"
```

Comment:
oops, I just realized that I forgot to re-branch off master. So this PR included the work I did for #8. :grin: